{"schemaVersion":{"patch":0,"minor":3,"major":0},"metadata":{"role":"symbol","extendedModule":"Swift","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"localizedDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"modules":[{"relatedModules":["Swift"],"name":"SwiftKyber"}],"platforms":[{"deprecated":false,"introducedAt":"8.0","unavailable":false,"beta":false,"name":"iOS"},{"name":"macOS","deprecated":false,"unavailable":false,"beta":false,"introducedAt":"10.10"},{"deprecated":false,"introducedAt":"9.0","unavailable":false,"beta":false,"name":"tvOS"},{"introducedAt":"2.0","beta":false,"deprecated":false,"unavailable":false,"name":"watchOS"}],"roleHeading":"Instance Property","externalID":"s:s5ErrorP10FoundationE20localizedDescriptionSSvp::SYNTHESIZED::s:10SwiftKyber0B9ExceptionO","title":"localizedDescription","symbolKind":"property"},"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"localizedDescription"},{"kind":"text","text":": "},{"text":"String","kind":"typeIdentifier","preciseIdentifier":"s:SS"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}],"kind":"declarations"}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/localizedDescription"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/kyberexception\/localizeddescription"]}],"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"Error.localizedDescription"},{"type":"text","text":"."}],"sections":[],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/SwiftKyber\/documentation\/SwiftKyber","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/Error-Implementations"]]},"references":{"doc://SwiftKyber/documentation/SwiftKyber/KyberException/Error-Implementations":{"title":"Error Implementations","url":"\/documentation\/swiftkyber\/kyberexception\/error-implementations","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/Error-Implementations","abstract":[],"role":"collectionGroup","type":"topic","kind":"article"},"doc://SwiftKyber/documentation/SwiftKyber/KyberException/localizedDescription":{"url":"\/documentation\/swiftkyber\/kyberexception\/localizeddescription","title":"localizedDescription","abstract":[],"identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/localizedDescription","type":"topic","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"localizedDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"}],"kind":"symbol","role":"symbol"},"doc://SwiftKyber/documentation/SwiftKyber/KyberException":{"title":"KyberException","type":"topic","url":"\/documentation\/swiftkyber\/kyberexception","role":"symbol","kind":"symbol","abstract":[{"type":"text","text":"The Kyber exceptions"}],"identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"KyberException"}],"navigatorTitle":[{"text":"KyberException","kind":"identifier"}]},"doc://SwiftKyber/documentation/SwiftKyber":{"abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}],"url":"\/documentation\/swiftkyber","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber","title":"SwiftKyber","role":"collection","type":"topic","kind":"symbol"}}}